cloud.foundry.cli/src/test/resources/demo/diffSpaceDevs.yml

```diff
@@ -0,0 +1,3 @@
+spaceDevelopers:
+  - dummy.1@test.org
+  - dummy.2@test.org
```

Comment:
A newline could be added to the end of the file

Reply:
Deleted this file, since we don't really need this file. We will have an example file with the Collect YML Files US.